packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -112,7 +112,7 @@ const isIconStyles = ( {
 	};
 
 	return css`
-		color: ${ COLORS.gray[ 900 ] };
+		color: ${ COLORS.theme.foreground };
```

Comment:
This actually fixes active state for toggles with icon:

|Before|After|
|---|---|
|![Screenshot 2024-12-09 at 11 54 52](https://github.com/user-attachments/assets/e874f3c1-3b64-496f-aa01-123d0f876e8f)|![Screenshot 2024-12-09 at 11 54 44](https://github.com/user-attachments/assets/b22e7e00-6825-4985-a2ec-97055085ed00)|